types/truffle-privatekey-provider/tslint.json

```diff
@@ -0,0 +1,80 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
I don't think you need these rule exemptions. `tslint.json` should just be `{ "extends": "dtslint/dt.json" }`.